api.md

```diff
@@ -2029,51 +2109,57 @@ background worker checks the policy. If you need to change the data retention
 interval or reorder by a different index, you'll need to remove the policy and
 add a new one.
 
-#### Required Arguments [](alter_job_schedule-required-arguments)
+#### Required Arguments [](alter_job-required-arguments)
 
 |Name|Description|
 |---|---|
 | `job_id` | (INTEGER) the id of the policy job being modified |
 
 
-#### Optional Arguments [](alter_job_schedule-optional-arguments)
+#### Optional Arguments [](alter_job-optional-arguments)
 
 |Name|Description|
 |---|---|
 | `schedule_interval` | (INTERVAL)  The interval at which the job runs |
 | `max_runtime` | (INTERVAL) The maximum amount of time the job will be allowed to run by the background worker scheduler before it is stopped |
 | `max_retries` | (INTEGER)  The number of times the job will be retried should it fail |
 | `retry_period` | (INTERVAL) The amount of time the scheduler will wait between retries of the job on failure |
-| `if_exists` | (BOOLEAN)  Set to true to avoid throwing an error if the job does not exist, a notice will be issued instead. Defaults to false. |
+| `scheduled` | (BOOLEAN)  Set to `FALSE` to exclude this job from being run as background job. |
```

Comment:
For all boolean optional arguments, specify the default?  (It does on some but not all)

Reply:
This is optional arguments section where default is NULL